### ISSUE TITLE
Taiko drum sampleset is completely silent

### ISSUE THREAD

**Issue by nl-tatatat** [ppy/osu-stable-issues#67]
The default drum sampleset for the taiko gamemode is completely silent, giving no player feedback. While most people don't use this sampleset in taiko maps (because its unrankable because of this issue), its a very common occurrence in converted difficulties.
 Its very hard to follow the beat when you have no feedback.

**Comment by peppy** [comment]
Fixed in next cutting edge.
